src/databricks/labs/ucx/source_code/known.json

```diff
@@ -7323,6 +7323,278 @@
     "pydevd_plugins.extensions": [],
     "pydevd_plugins.extensions.pydevd_plugin_omegaconf": []
   },
+  "openai": {
```

Comment:
@ericvergnaud : OpenAI probably does not work with Spark. Does it make sense to include it in that case?

Reply:
Or maybe only the top level to avoid linting?

Reply:
It's here because it's used in solacc.
There's no concept of 'top-level'. If you `import openai.cli`, ucx looks for openai.cli, not for openai. The top-level key is the name of the distribution, not of a module.